spec/controllers/api/agencies_controller_spec.rb

```diff
@@ -78,6 +78,7 @@ def expected_response
                 {
                   id: event_date.id,
                   event_id: event.id,
+                  capacity: 25,
```

Comment:
where does the test capacity get set?

Reply:
It's the default for the factory. https://github.com/midohiofoodbank/freshtrak-pantry-finder-api/blob/612aed7c1d6031847c11b5c0c8b535ca46251553/spec/factories/event_date.rb#L10